### PR TITLE
Boardrun Classic & Bizarre layouts

### PR DESCRIPTION
**Pull request by nopunin10did**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by nopunin10did** [comment]
@lukeski14 Created the PR

**Comment by lukeski14** [comment]
This is a more complete and accurate implementation of the two Boardrun PCBs. I recommend merging this PR and closing #10479.

File: keyboards/boardrun/classic/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     * |        |        | MENU   |        | RESET  |        |        |        |        |        |        | INS    |        | PRT SC |        |
+     * |--------+--------+--------+--------+--------+--------+--------+        +--------+--------+--------+--------+--------+--------+--------|
+     * | CAPSLK |        | SCROLL |        |        |        |        |        |        |        |        |        |        |        | RCTL   |
+     * |--------+--------+--------+--------+--------+--------+--------+        +--------+--------+--------+--------+--------+--------+--------|
+     * | UK \|  |        |        |        |        | PAUSE  |        |        |        |        | MUTE   | VOL DN | VOL UP |        |        |
+     * '--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+--------+--------+--------+--------'
+     *          | RGUI   |        |        | RALT   |                 |        |                 |        |        |        |        |
+     *          '--------------------------------------------------------------------------------------------------------------------'
+     */
+    [_FNCLASSIC] = LAYOUT_CLASSIC(
```
Comment (drashna) [inline keyboards/boardrun/classic/keymaps/default/keymap.c:66]:
```suggestion
    [_FNCLASSIC] = LAYOUT_classic(
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by nopunin10did** [comment]
@drashna , all requested changes were made.

**Comment by lukeski14** [comment]
@nopunin10did I don't think you made all the changes requested, 10 still remain which have caused the build to fail

**Comment by nopunin10did** [comment]
> @nopunin10did I don't think you made all the changes requested, 10 still remain which have caused the build to fail

That's odd.  I'll dig through the requests to see what may have been missed.

**Comment by nopunin10did** [comment]
> @nopunin10did I don't think you made all the changes requested, 10 still remain which have caused the build to fail

Ugh, figured it out.  GitHub's "Commit these changes" button can cause other changes requested to be marked as resolved without actually fixing them.  Doing so now.

**Comment by nopunin10did** [comment]
Okay, @lukeski14 and @drashna , _now_ all the changes should be in place.

File: keyboards/boardrun/bizarre/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The Boardrun is an evolution of the Boardwalk layout. The Boardwalk proved that the Ergodox keycap set could be adapted to more keyboards that provided a more compact design and layout flexibility. While adaptation was a design goal of the Boardwalk layout, the Boardrun was designed to distill and refine the best aspects of the current state of keyboard layouts, taking into account current trends in staggered design, the Ergodox keycap layout, and the Boardwalk's most popular layouts. To that end, a columnar staggered, ortholinear layout with alpha rotation was settled on, using the "HHKB" layout from the Boardwalk. The result is two layouts that are comfortable, familiar, and have great compatibility with keysets.
+
+-   Keyboard Maintainer: QMK Community
+-   Hardware Supported: Boardrun Bizarre Ortholinear PCB
+-   Hardware Availability: mkultra.click
+
+Make example for this keyboard (after setting up your build environment):
+
```
Comment (Erovia) [inline keyboards/boardrun/bizarre/readme.md:14]:
```suggestion

    make boardrun/bizarre:default

Flashing example for this keyboard:

    make boardrun/bizarre:default:flash
```

File: keyboards/boardrun/classic/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+The Boardrun is an evolution of the Boardwalk layout. The Boardwalk proved that the Ergodox keycap set could be adapted to more keyboards that provided a more compact design and layout flexibility. While adaptation was a design goal of the Boardwalk layout, the Boardrun was designed to distill and refine the best aspects of the current state of keyboard layouts, taking into account current trends in staggered design, the Ergodox keycap layout, and the Boardwalk's most popular layouts. To that end, a columnar staggered, ortholinear layout with alpha rotation was settled on, using the "HHKB" layout from the Boardwalk. The result is two layouts that are comfortable, familiar, and have great compatibility with keysets.
+
+-   Keyboard Maintainer: QMK Community
+-   Hardware Supported: Boardrun Classic Ortholinear PCB
+-   Hardware Availability: mkultra.click
+
+Make example for this keyboard (after setting up your build environment):
+
```
Comment (Erovia) [inline keyboards/boardrun/classic/readme.md:14]:
```suggestion

    make boardrun/classic:default

Flashing example for this keyboard:

    make boardrun/classic:default:flash
```

File: keyboards/boardrun/classic/rules.mk [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = yes        # Unicode
```
Comment (Erovia) [inline keyboards/boardrun/classic/rules.mk:23]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
UNICODE_ENABLE = no         # Unicode
```

Bootmagic Lite is recommended.
Unless there is a specific need, I'd leave the command and unicode functionality out by default to save space.

File: keyboards/boardrun/classic/readme.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    make boardrun/classic:default:flash
+
```
Comment (Erovia) [inline keyboards/boardrun/classic/readme.md:16]: Can you include "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

File: keyboards/boardrun/bizarre/readme.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    make boardrun/bizarre:default:flash
+
```
Comment (Erovia) [inline keyboards/boardrun/bizarre/readme.md:16]: Can you include "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

File: keyboards/boardrun/bizarre/rules.mk [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = yes        # Unicode
```
Comment (Erovia) [inline keyboards/boardrun/bizarre/rules.mk:23]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
UNICODE_ENABLE = no         # Unicode
```

Bootmagic Lite is recommended.
Unless there is a specific need, I'd leave the command and unicode functionality out by default to save space.

**Review by Erovia — CHANGES_REQUESTED** [review]

File: keyboards/boardrun/bizarre/readme.md (hunk 2 above)
Comment (nopunin10did) [reply keyboards/boardrun/bizarre/readme.md:14]: Change made as requested in latest commit.

File: keyboards/boardrun/classic/rules.mk (hunk 4 above)
Comment (nopunin10did) [reply keyboards/boardrun/classic/rules.mk]: Change made as requested in latest commit.

File: keyboards/boardrun/bizarre/rules.mk (hunk 7 above)
Comment (nopunin10did) [reply keyboards/boardrun/bizarre/rules.mk:23]: Change made as requested in latest commit.

File: keyboards/boardrun/bizarre/readme.md (hunk 6 above)
Comment (nopunin10did) [reply keyboards/boardrun/bizarre/readme.md:22]: Change made as requested in latest commit.

File: keyboards/boardrun/classic/readme.md (hunk 5 above)
Comment (nopunin10did) [reply keyboards/boardrun/classic/readme.md:20]: Change made as requested in latest commit.

File: keyboards/boardrun/classic/readme.md (hunk 3 above)
Comment (nopunin10did) [reply keyboards/boardrun/classic/readme.md:14]: Change made as requested in latest commit.

**Comment by nopunin10did** [comment]
@Erovia , the changes you requested were made.  Our Travis build keeps failing due to timeout; is this a widespread issue?  I have an unrelated PR where the same thing is occurring.

**Comment by fauxpark** [comment]
@nopunin10did the issue is that Travis is looking at the difference between this branch and current master, which has moved on since you branched off of it. If you rebase, it should build only these boards.

**Comment by nopunin10did** [comment]
@fauxpark , I think I did the rebase wrong.  The branch seems to be borked, and git keeps complaining about authentication when I try to do a 
`git push --force`
after resetting to the prior clean commit.

**Pull request closed without merging**